extra-web/seamonkey/autobuild/patches/1000-seamonkey-2.53.7-mozilla-440908.patch

```diff
@@ -0,0 +1,92 @@
+--- mozilla/modules/libpref/Preferences.cpp.orig	2021-03-01 21:17:57.000000000 +0300
```

Comment:
Please, mark origin of these patches (Fedora) in the file name, `1000-Fedora-...` and etc.